components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/java/org/wso2/carbon/identity/entitlement/ui/util/PolicyCreatorUtil.java

```diff
@@ -35,6 +35,8 @@
  * This is Util class which help to create a XACML policy
```

Comment:
@AymanDF Need to remove commented out code also.
@johannnallathamby All logic in this class seems to have commented out, shall we remove the class?